VascularFlow/Numerics/ArrayFirstDerivative.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+def array_first_derivative(array: np.ndarray, dx_e: float) -> np.ndarray:
```

Comment:
What does this function do? Please add a documentation string.

It looks like some finite differences scheme to me.